holoviews/plotting/bokeh/tabular.py

```diff
@@ -19,6 +19,17 @@ class TablePlot(BokehPlot, GenericElementPlot):
 
     height = param.Number(default=None)
 
+    finalize_hooks = param.HookList(default=[], doc="""
+        Optional list of hooks called when finalizing a column
+        (deprecated in favor of hooks). The hook is passed the plot
+        object and the displayed object, and other plotting handles
+        can be accessed via plot.handles.""")
```

Comment:
Seems like the docstring here should simply be "Deprecated; use `hooks` instead.".